DigitalLearningSolutions.Data.Migrations/Scripts/UAR-831-CreateViewsForAdminUsersAndCandidatesTables-UP.sql

```diff
@@ -1,45 +1,43 @@
+SELECT dbo.AdminAccounts.AdminID,
+       null                                    AS Login,
+       dbo.Users.PasswordHash                  AS Password,
        dbo.AdminAccounts.CentreID,
-       dbo.AdminAccounts.IsCentreAdmin                   AS CentreAdmin,
-       dbo.AdminAccounts.ConfigAdmin_deprecated          AS ConfigAdmin,
-       dbo.AdminAccounts.IsReportsViewer                 AS SummaryReports,
-       dbo.AdminAccounts.IsSuperAdmin                    AS UserAdmin,
-       dbo.Users.FirstName                               AS Forename,
-       dbo.Users.LastName                                AS Surname,
-       dbo.AdminAccounts.Email,
+       dbo.AdminAccounts.IsCentreAdmin         AS CentreAdmin,
+       0                                       AS ConfigAdmin,
+       dbo.AdminAccounts.IsReportsViewer       AS SummaryReports,
+       dbo.AdminAccounts.IsSuperAdmin          AS UserAdmin,
+       dbo.Users.FirstName                     AS Forename,
+       dbo.Users.LastName                      AS Surname,
+       dbo.Users.PrimaryEmail                  AS Email,
        dbo.AdminAccounts.IsCentreManager,
-       dbo.AdminAccounts.Approved_deprecated             AS Approved,
-       dbo.AdminAccounts.PasswordReminder_deprecated     AS PasswordReminder,
-       dbo.AdminAccounts.EITSProfile_deprecated          AS EITSProfile,
-       dbo.AdminAccounts.PasswordReminderHash_deprecated AS PasswordReminderHash,
-       dbo.AdminAccounts.PasswordReminderDate_deprecated AS PasswordReminderDate,
+       true                                    AS Approved,
+       0                                       AS PasswordReminder,
+       ""                                      AS EITSProfile,
+       null                                    AS PasswordReminderHash,
+       null                                    AS PasswordReminderDate,
        dbo.AdminAccounts.Active,
-       dbo.AdminAccounts.IsContentManager                AS ContentManager,
+       dbo.AdminAccounts.IsContentManager      AS ContentManager,
        dbo.AdminAccounts.PublishToAll,
        dbo.AdminAccounts.ImportOnly,
-       dbo.Users.TermsAgreed                             AS TCAgreed,
-       dbo.AdminAccounts.IsContentCreator                AS ContentCreator,
+       dbo.Users.TermsAgreed                   AS TCAgreed,
+       dbo.AdminAccounts.IsContentCreator      AS ContentCreator,
        dbo.Users.FailedLoginCount,
        dbo.Users.ProfileImage,
-       dbo.AdminAccounts.IsSupervisor                    AS Supervisor,
-       dbo.AdminAccounts.IsTrainer                       AS Trainer,
-       dbo.AdminAccounts.CategoryID,
-       dbo.AdminAccounts.SkypeHandle_deprecated          AS SkypeHandle,
-       dbo.AdminAccounts.PublicSkypeLink_deprecated      AS PublicSkypeLink,
+       dbo.AdminAccounts.IsSupervisor          AS Supervisor,
+       dbo.AdminAccounts.IsTrainer             AS Trainer,
+       ISNULL(dbo.AdminAccounts.CategoryID, 0),

```

Comment:
I think this is going to need to be aliased as "AS CategoryID" as the ISNULL does not automatically name its output.

Reply:
Ah right, done :)